masonite/view.py

```diff
@@ -51,3 +68,76 @@ def extend(self):
     def share(self, dictionary):
         self.dictionary.update(dictionary)
         return self
+
+    def cache_for(self, time=None, type=None):
+        """
+        Set time and type for cache
+        """
+        self.cache = True
+        self.cache_time = float(time)
+        self.cache_type = type
+        if self.__is_expired_cache():
+            self.__create_cache_template(self.template)
+        return self
+
+    def __create_cache_template(self, template):
+        """
+        Save in the cache the template
+        """
+
+        cache_type = self.cache_type.lower()
+        calc = 0
+        if cache_type == "second" or cache_type == "seconds":
```

Comment:
Try out something here. This looks really ugly and I'm trying to figure out how to make this cleaner. Possibly change:

```python
if cache_type == "second" or cache_type == "seconds":
```

to

```python
if cache_type in ("second", "seconds"):
```

That should make it look less ugly, shorter, and more readable